backfill/2020-03-30-gcs-error/append_results_structured.sh

```diff
@@ -0,0 +1,68 @@
+#!/bin/bash
+
+PROJECT=moz-fx-data-backfill-30
+
+set -x
+
+function make_query {
+    local dataset=$1 
+    local table=$2 
+    local start_ds=$3
+    local end_ds=$4
+    cat << EOF
+WITH seen_documents AS (
+    SELECT DISTINCT
+        document_id
+    FROM
+        \`moz-fx-data-shared-prod\`.${dataset}.${table}
+    WHERE
+        date(submission_date) >= "${start_ds}"
+        AND date(submission_date) <= "${end_ds}"
```

Comment:
To more closely match exactly how copy_dedupe works, I think we should limit this query to a single date and run a separate query job per target date.

Reply:
Can appends be done in bulk across a range of dates without having to specify a specific partition id? If so, I'll just add the submission date as a partition in the window function for deduping above.